api-server/test/routes/session_test.py

```diff
@@ -1,12 +1,17 @@
 import pytest
-from httpx import AsyncClient
+from httpx import AsyncClient, Cookies
+from services.auth import Auth
 from models import Session
 
 
 @pytest.mark.usefixtures("clear_db")
 class TestMakeSession:
     @pytest.mark.anyio
     async def test_make_session(self, api_client: AsyncClient):
+        cookies = Cookies()
+        username = "test_username"
+        cookies.set("token", Auth.encode(username))
```

Comment:
Should I mock `Auth.encode`? How do I do that if I want to mock it?

Reply:
I think this is fine. I would add a test that ensures that the `/session/make` endpoint returns a 401 if the cookie doesn't exist or is malformed.